satellite/satellitedb/migrate.go

```diff
@@ -530,14 +530,36 @@ func (db *DB) PostgresMigration() *migrate.Migration {
 					`ALTER TABLE bucket_bandwidth_rollups ADD CONSTRAINT bucket_bandwidth_rollups_pk PRIMARY KEY (bucket_name, project_id, interval_start, action);`,
 				},
 			},
+			{
+				Description: "Add new Columns to store version information",
+				Version:     14,
+				Action: migrate.SQL{
+					`ALTER TABLE nodes ADD major bigint;
```

Comment:
You can do `ALTER TABLE nodes ADD major BIGINT NOT NULL DEFAULT 0` as a single statement instead of doing `ADD`, `UPDATE` and `ALTER COLUMN`.